hiss/user/admin.py

```diff
@@ -12,6 +15,20 @@ def check_in(_modeladmin, _request, queryset) -> None:
     queryset.update(checked_in_at=timezone.datetime.now())
 
 
+def export_emails(_modeladmin, _request, queryset):
```

Comment:
Try and add as many type hints as you can to functions. Also, this name conflicts with the other `export_emails` function in `application`.

```suggestion
from django.db.models.query import QuerySet
# ...

def export_user_emails(_modeladmin, _request: HttpRequest, queryset: QuerySet) -> HttpResponse:
```